### PR TITLE
Give some flexibilty around filters and outputs

### PR DESCRIPTION
**Pull request by yasn77**
- Create option to add additional outputs and filters

- Give ability for user to disable certain outputs
   This is useful for when the user only wants certain logs (i.e they don't
   need container logs or only want k8s audit)

### CODE REVIEW

File: helm/fluent-logshipping-app/templates/configmap.yaml [hunk 1]
```diff
@@ -101,6 +101,11 @@ data:
         Skip_Long_Lines   ${input_skip_long_lines}
 
   filters.conf: |
+    {{- if .Values.additional_filters }}
```
Comment (QuentinBisson) [inline helm/fluent-logshipping-app/templates/configmap.yaml:104]: Sorry to be pedantic but would extra_filters not be best here?

Also, why use underscores when we use camel cases in other places like `Values.outputs.elasticsearch.inputLogTypes`

File: helm/fluent-logshipping-app/values.yaml [hunk 2]
```diff
@@ -48,6 +54,7 @@ outputs:
   azure:
     logAnalytics:
       enabled: false
+      inputLogTypes: *inputTypes
```
Comment (QuentinBisson) [inline helm/fluent-logshipping-app/values.yaml:57]: Should we not name it logTypes instead?

Comment (yasn77) [reply helm/fluent-logshipping-app/values.yaml:57]: I am OK with `logTypes`, Just in my head at the time I was thinking I am matching against the inputs

File: helm/fluent-logshipping-app/templates/configmap.yaml (hunk 1 above)
Comment (yasn77) [reply helm/fluent-logshipping-app/templates/configmap.yaml:104]: Sure, I'll update... It's better for consistency

**Review by QuentinBisson — APPROVED** [review]
LGTM

**Pull request merged**